File: C#/ConsolePr5.cs
class Program
{
    static void Main()
    {
        int number, stepen1, stepen2;
        
        Console.Write("Number: ");
        number = int.Parse(Console.ReadLine());
        
        Console.Write("pow of Number: ");
        stepen1 = int.Parse(Console.ReadLine());
        
        Console.Write("pow of pow: ");
        stepen2 = int.Parse(Console.ReadLine());
        
        long result = (long)Math.Pow(number, stepen1);
        result = (long)Math.Pow(result, stepen2);
        
        int lastfigure = (int)(result % 10);
        
        Console.WriteLine("the last digit of number: " + lastfigure);
    }
}

File: ConsolePR5.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    int number, stepen1, stepen2;
    
    cout << "Number: ";
    cin >> number;

    cout << "pow of Number: ";
    cin >> stepen1;

    cout << "pow of pow: ";
    cin >> stepen2;

    long result = pow(number, stepen1);
    result = pow(result, stepen2);

    int lastfigure = result % 10;

    cout << "the last digit of number: " << lastfigure << endl;

    return 0;
}